var/spack/repos/builtin/packages/brion/package.py

```diff
@@ -30,6 +31,8 @@ class Brion(CMakePackage):
     depends_on('boost +shared', when='~python')
     depends_on('boost +shared +python', when='+python')
 
+    depends_on('libsonata', when='@3.1.0')
```

Comment:
```suggestion
    depends_on('libsonata', when='@3.1.0:')
```
I assume this dependency will stay in the future. Then we should use an open range (: